static/templates/settings/dropdown_list_widget.hbs

```diff
@@ -17,7 +17,9 @@
             </ul>
         </span>
     </label>
+    {{#if reset_button_text}}
     <button class="button small dropdown_list_reset_button">
         <a>{{reset_button_text}}</a>
     </button>
+    {{/if}}
```

Comment:
This should probably be pulled out as a separate preparatory fix for `dropdown_list_widget`, I think.